ukis_csmask/mask.py

```diff
@@ -52,6 +54,13 @@ def __init__(
         if img.dtype != np.float32:
             raise TypeError("img must be in top of atmosphere reflectance with dtype float32")
 
+        if img.shape[0] != 256 or img.shape[1] != 256:
```

Comment:
This should read `if img.shape[0] < 256 or img.shape[1] < 256:`. We need to check if the input image size is smaller than the internal tiling size. If the input image is larger or equal 256 no warning should be raised.